Zend/zend_compile.h

```diff
@@ -266,6 +266,9 @@ typedef struct _zend_oparray_context {
 /* Children must reuse parent get_iterator()              |     |     |     */
 #define ZEND_ACC_REUSE_GET_ITERATOR      (1 << 17) /*  X  |     |     |     */
 /*                                                        |     |     |     */
+/* Class is declared "locked"                             |     |     |     */
+#define ZEND_ACC_LOCKED                  (1 << 18) /*  X  |     |     |     */
```

Comment:
Since this constant will be exposed to userland, whereas most of these aren't, such a high number might seem surprising to users, but I'm not sure of the impact of rearranging this list.

Reply:
We don't make any particularly guarantees about the values of those flags (in fact, they change between releases). I wouldn't worry about this.